core/src/main/java/org/axonframework/commandhandling/model/inspection/AnnotatedAggregate.java

```diff
@@ -458,8 +458,8 @@ public Object handle(CommandMessage<?> msg) throws Exception {
         if (aggregateRoot != null) {
             deadlineManager.schedule(triggerDateTime, deadlineContext(), deadlineInfo, scheduleToken);
         } else {
-            delayedTasks.add(() -> deadlineManager
-                    .schedule(triggerDateTime, deadlineContext(), deadlineInfo, scheduleToken));
+            delayedTasks.add(DelayedTask.lowPriorityTask(() -> deadlineManager
```

Comment:
You schedule the deadline calls lowest as you want to ensure any prior events which might set the aggregate identifier/other state have been handled, right?

Reply:
Exactly! We need to be sure that this aggregate has an Identifier assigned to itself so we can schedule a deadline. Otherwise, the deadline manager will not know which aggregate to invoke.